OracleDatabase/SingleInstance/helm-charts/oracle-db/README.md

```diff
@@ -80,6 +80,7 @@ The following tables lists the configurable parameters of the Oracle  Database c
 | image                                | Image to pull                              | container-registry.oracle.com/database/enterprise:19.3.0.0 |
 | imagePullPolicy                      | Image pull policy                          | Always                                                     |
 | imagePullSecrets                     | container registry login/password          |                                                            |
+| enable_archive                       | Set true to enable archive mode in database| false                                                      |
```

Comment:
Why is this enable_archive but the variable ENABLE_ARCHIVELOG ?

Reply:
Just wanted to differentiate between helm option and the environment variable, that's why used the name enable_archive.
But I also feel now it should be enable_archivelog for consistency. So, changed it to enable_archivelog.